tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -359,6 +360,16 @@ netebpfext_unit_invoke_sock_addr_program(
     auto client_context = (test_sock_addr_client_context_t*)client_binding_context;
     auto sock_addr_context = (bpf_sock_addr_t*)context;
     int action;
+    int32_t is_admin = 0;
+
+    auto sock_addr_extension_data =
```

Comment:
Only use auto when the type is obvious from the right side of the `=` (like line 367 is fine)

Reply:
Thanks. Fixed.